### ISSUE TITLE
[Feature Request] Support Cloud provider for Huawei

### ISSUE THREAD

**Issue by energy162** [cnrancher/autok3s#611]
RT

**Comment by JacieChao** [comment]
Yes. We have considered adding several popular cloud providers for AutoK3s in progress. If more people need it, we will consider raising the priority of it.
And we are also happy to accept community contributions about different cloud providers.
